extra/setup.py

```diff
@@ -46,7 +46,7 @@ def create_version_file():
 
 requirements = [
     'numpy==1.19.5',
-    'gluoncv>=0.9.2,<1.0',
+    'gluoncv>=0.10.0,<1.0',
```

Comment:
Since gluoncv depends on autogluon.core, we will probably need to cap the version of gluoncv for v0.1 release to ensure that we are using a version of gluoncv that is compatible with `autogluon.core==0.1`, and then we can uncap it after v0.1 release.

For example for v0.1 release:

```
'gluoncv>=0.10.0,<0.11'
```

And then we can change to

```
'gluoncv>=0.10.0,<1.0',
```

for nightly pre-release after v0.1 is released.

What do you think?